backend/src/services/new-cars/new-cars.service.ts

```diff
@@ -1,8 +1,8 @@
 import { prisma } from '@data/prisma-client';
 
-import type { ModelResponseDto } from '@autoline/shared/common/types/types';
+import type { CarPreviewType } from '@autoline/shared/common/types/types';
 
-const getNewCars = async (limit: number): Promise<ModelResponseDto[]> => {
+const getNewCars = async (limit: number): Promise<CarPreviewType[]> => {
```

Comment:
It's not the best practice to name type in the TS as `SmthType` because it's a tautology :slightly_smiling_face: